resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -584,6 +595,128 @@ class TextCursor
     }
 }
 
+[DSCResource()]
+class ColorFilter
+{
+    # Key required. Do not set.
+    [DscProperty(Key)] [string] $SID
+    [DscProperty()] [nullable[bool]] $FilterStatus
+    [DscProperty()] [ColorFilters] $FilterColor
+    [DscProperty()] [nullable[bool]] $KeyboardShortcutStatus
+
+    static hidden [string] $FilterStatusProperty = 'Configuration'
+    static hidden [string] $FilterStatusActiveProperty = 'Active'
+    static hidden [string] $FilterStatusValue = 'colorfiltering'
+    static hidden [string] $FilterColorProperty = 'FilterType'
+    static hidden [string] $KeyboardShortcutStatusProperty = 'HotkeyEnabled'
+
+    static [bool] GetStatus()
+    {
+        $FilterStatusArgs = @{  Path = $global:NTAccessibilityRegistryPath; Name = ([ColorFilter]::FilterStatusProperty)}
+        if (-not(DoesRegistryKeyPropertyExist @FilterStatusArgs))
+        {
+            return $false
+        }
+        else
+        {
+            $colorFilterSetting = (Get-ItemProperty @FilterStatusArgs).Configuration
+            return ($colorFilterSetting -eq ([ColorFilter]::FilterStatusValue))

```

Comment:
Shouldn't this be just checking whether the active key is set to 0 or 1?

Reply:
This is because I wasn't clear on what I was doing when I wrote this. It's half of one setting and half of another. https://github.com/microsoft/winget-dsc/pull/66#discussion_r1802071410